dashboard_desc.txt

```diff
@@ -0,0 +1,6 @@
+For building the dashboard, I liked the mockup given and decided to build the project off it. I wanted a minimalist design and so I stuck with a two-tone background, I choose a light green, in homage to Sidekick's green tone.
+
+I used bootstrap 4.0 for their powerful flexbox layout. Each course is it's own row and the credit squares are nested flexboxes.
+In things I could further improve, I would use embedded js to use loops to prevent repition in code. Also I would display the data from response-data.json rather than hardcode it. 
```

Comment:
:+1: How would you plan to prep and prepare the data?

Reply:
Assuming the back-end for course enrollment was setup, I'd create a `Student` object which would include nested `course` objects. Each `course` would include a credits field which I would query from to get the required data. There's also the security requirement regarding who has access to read/write to the database.